src/components/pipelineRuns-details/PipelineRunLogs.tsx

```diff
@@ -263,7 +267,20 @@ type PipelineRunLogsWithActiveTaskProps = {
   obj: PipelineRunKind;
 };
 
-const PipelineRunLogs = withTranslation()(PipelineRunLogsWithTranslation);
+const TranslatedPipelineRunLogs = withTranslation()(
+  PipelineRunLogsWithTranslation,
+);
+
+const PipelineRunLogs = (props) => {
+  const isDevConsoleProxyAvailable = useFlag(FLAGS.DEVCONSOLE_PROXY);
+
+  return (
+    <TranslatedPipelineRunLogs
+      {...props}
+      isDevConsoleProxyAvailable={isDevConsoleProxyAvailable}
```

Comment:
Are you making use of this prop `isDevConsoleProxyAvailable`?

Reply:
Updated the PR.